### DISCUSSION TITLE
CroxyProxy, i'm desperate.

### DISCUSSION THREAD

**Discussion by funkymokey** [3kh0/ext-remover#731] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm trying to download a few files for a project I'm doing, i have no USB stick or any way to transfer files from one pc to another, I do remember that CroxyProxy was able to download files correctly without it coming out as a gxg. If you have a CroxyProxy link or few HMU pls

**Comment by doofenator** [comment, marked as answer]
https://167.71.43.43/
heres a croxy proxy
